acra-censor/handlers/querycapture_handler.go

```diff
@@ -12,9 +12,12 @@ import (
 	"bytes"
 	"github.com/cossacklabs/acra/logging"
 	log "github.com/sirupsen/logrus"
+	"github.com/xwb1989/sqlparser"
+	"github.com/xwb1989/sqlparser/dependency/querypb"
 )
 
 const DefaultSerializationTimeout = time.Second
+const ValuePlaceholder = "X"
```

Comment:
better to change to some more obvious as placeholder, something like `::replaced::`
because I think sometimes in the future even me will be surprised during debugging when will see this value when it can be by logic ))